src/Model/Model_System_Report.php

```diff
@@ -285,19 +296,30 @@ protected function prepare_report( $item ): array {
 			'label'        => $item['label'],
 			'label_export' => $item['label'],
 			'value'        => $item['value'],
-			'value_export' => $item['value'],
+			'value_export' => ( $item['value_export'] !== '' ) ?: $item['value'],
 		];
 	}
 
 	/**
 	 * @since 6.0
 	 */
-	protected function is_temporary_folder_writable(): string {
+	protected function is_temporary_folder_writable(): array {
 		$is_writable = wp_is_writable( $this->data->mpdf_tmp_location );
 
-		$output  = $is_writable ? __( 'Writable', 'gravityforms' ) : __( 'Not writable', 'gravityforms' );
-		$output .= $this->getController()->view->get_icon( $is_writable );
+		$string  = $is_writable ? __( 'Writable', 'gravityforms' ) : __( 'Not writable', 'gravityforms' );
+		$icon = $this->getController()->view->get_icon( $is_writable );
+
+		return ['value' => $string . $icon, 'value_export' => $string ];
```

Comment:
`$string` is still translated when passed to `value_export`. Will need to hardcode the strings instead.